README.md

```diff
@@ -174,6 +174,17 @@ illuminatio 1.1 is confirmed to be working properly with the following kubernete
 - Google Kubernetes Engine, v1.12.8-gke.10
 - kubeadm 1.15.0-00, kubernetes v1.15.2
 
+### PodSecurityPolicy
+
+If your cluster has the [PodSecurityPolcy](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podsecuritypolicy) Admission Controller you must ensure that illuminatio has the following rights to be created:
```

Comment:
Typo:

```suggestion
If your cluster has the [PodSecurityPolicy](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podsecuritypolicy) Admission Controller you must ensure that illuminatio has the following rights to be created:
```

Reply:
Maybe also clarify, that the runner requires those privileges (the CLI doesn't):

```suggestion
If your cluster has the [PodSecurityPolicy](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podsecuritypolicy) Admission Controller you must ensure that the illuminatio runner has the following rights to be created:
```